sdk/python/feast/feature_view_projection.py

```diff
@@ -41,7 +41,7 @@ def name_to_use(self):
     def to_proto(self) -> FeatureViewProjectionProto:
         feature_reference_proto = FeatureViewProjectionProto(
             feature_view_name=self.name,
-            feature_view_name_alias=self.name_alias or "",
+            feature_view_name_alias=self.name_alias or None,
```

Comment:
I guess we have our answer now 😄 this was probably due to mypy checks and also because protos don't really distinguish between null and empty string anyway. Maybe try changing `from_proto` method instead to populate the object with `None` even if `name_alias` is set to `""`, empty string for a name doesn't make sense anyway.